libs/libsemanage/Makefile

```diff
@@ -0,0 +1,100 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libsemanage
+PKG_VERSION:=3.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/SELinuxProject/selinux/releases/download/20200710
+PKG_HASH:=22d6c75526e40d1781c30bcf29abf97171bdfe6780923f11c8e1c76a75a21ff8
+PKG_BUILD_DEPENDS:=PACKAGE_python-libsemanage:python3 PACKAGE_python-libsemanage:swig/host
+HOST_BUILD_DEPENDS:=audit/host libselinux/host bzip2/host
+
+PKG_MAINTAINER:=Thomas Petazzoni <thomas.petazzoni@bootlin.com>
+
+include $(INCLUDE_DIR)/package.mk
+include ../../lang/python/python3-package.mk
+
+define Package/libsemanage
+  SECTION:=libs
+  DEPENDS:=+libaudit +libselinux +libbz2
+  CATEGORY:=Libraries
+  TITLE:=SELinux policy management library
+  URL:=http://selinuxproject.org/page/Main_Page
+endef
+
+define Package/python-libsemanage
+  TITLE:=Python bindings for the SELinux policy management library
+  SUBMENU:=Python
+  SECTION:=lang
+  CATEGORY:=Languages
+  DEPENDS:=+python3 +libsemanage
+endef
+
+define Package/libsemanage/description
+	libsemanage is the policy management library. It uses
+	libsepol for binary policy manipulation and libselinux for
+	interacting with the SELinux system. It also exec's helper
+	programs for loading policy and for checking whether the
+	file_contexts configuration is valid (load_policy and
+	setfiles from policycoreutils) presently, although this may
+	change at least for the bootstrapping case (for rpm).
+endef #'
+
+include $(INCLUDE_DIR)/host-build.mk
+
+HOST_MAKE_FLAGS += \
+	DESTDIR=$(STAGING_DIR_HOSTPKG) \
+	PREFIX=""
+
+MAKE_FLAGS += \
+	PYTHON=$(PYTHON3) \
+        PYINC="-I $(PYTHON3_INC_DIR)" \
+        PYLIBS="-lpython$(PYTHON3_VERSION)" \
+
+ifdef CONFIG_PACKAGE_python-libsemanage
+  define Build/Compile/python-libsemanage
+        $(call Build/Compile/Default,swigify pywrap)
+  endef
+
+  define Build/Install/python-libsemanage
+        $(call Build/Install/Default,install-pywrap)
+  endef
+endif
+
+define Build/Configure
+endef
+
+define Build/Compile
+	$(call Build/Compile/Default,all)
+	$(Build/Compile/python-libsemanage)
+endef
+
+define Build/Install
+	$(call Build/Install/Default,install)
+	$(Build/Install/python-libsemanage)
+endef
+
+define Build/InstallDev
+	$(INSTALL_DIR) $(1)/usr/include
+	$(CP) $(PKG_INSTALL_DIR)/usr/include/* $(1)/usr/include/
+	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig
+	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/usr/lib/pkgconfig/libsemanage.pc $(1)/usr/lib/pkgconfig/
+	$(INSTALL_DIR) $(1)/usr/lib
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/* $(1)/usr/lib/
+endef
+
+define Package/libsemanage/install
+	$(INSTALL_DIR) $(1)/usr/lib
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libsemanage.so.* $(1)/usr/lib/
+endef
+
+$(eval $(call HostBuild))
+$(eval $(call BuildPackage,libsemanage))
```

Comment:
Please remove the non-Python part as it was moved to openwrt.git by https://github.com/openwrt/openwrt/commit/e8688098616edb64b0e75dc53e2dbe24c985c1ca